### ISSUE TITLE
The authentication source was not found when editing Sources

### ISSUE THREAD

**Issue by ricardoteixeiraduarte** [inverse-inc/packetfence#1551]
When editing an existing Source, the following error shows when saving the changes:

```
"The authentication source was not found"
```

**Comment by louismunro** [comment]
Can you tell which release you are running?

**Comment by rpryor** [comment]
Same issue here! Two clean installs of 6.2.1 on Debian 8.5. 
Add Active Directory as source works. Once you try to add rules and save then you get the "_Error! The authentication source was not found._"
Also tried to change the connection to bind to using ssl and port 636 which is known to be good and standard ldap 389 port. As well as all different scopes. None work. Inital bind looks good but cannot add rules.

**Comment by tribut** [comment]
We can reproduce this as well on both Packetfence 6.1.1 and 6.2.1 (Debian jessie).

It seems the form action is missing the id, both when editing the source and when editing a rule:

```
<form name="source" class="form-horizontal" method="post" action="https://login.stunet.tu-freiberg.de:1443/config/authentication/source//update">

<form name="rule" class="form form-horizontal" method="post" action="https://login.stunet.tu-freiberg.de:1443/config/authentication/source//rule//update">
```

**Comment by julsemaan** [comment]
@everybody

Is this still a bug you are experiencing in the latest and greatest (6.3) version ?

If so, we'll need to take a look at it.

**Comment by OddJobBob** [comment]
Confirmed this is still an issue in the latest 6.3 debian package.

**Comment by vinyanalista** [comment]
I can reproduce that using the latest Debian 8.6.0 and PacketFence 6.3.

**Comment by vinyanalista** [comment]
I could not reproduce that using the latest PacketFence 6.4.0 ZEN Edition (Zero Effort NAC) based on CentOS 6.8. Maybe an issue related with the distribution?

**Comment by jrouzierinverse** [comment]
This was an issue with only debian. 
It was fixed by commit db38b8e6085ae8c60906fd17d6cf5abf044c136e.
It was placed in maintenance.
